locale/en/download/package-manager.md

```diff
@@ -298,7 +298,7 @@ scoop install nodejs
 
 ## z/OS
 
-IBMŔ SDK for Node.js - z/OSŔ is available in two installation formats,
+IBMŔG SDK for Node.js - z/OSŔG is available in two installation formats,
 SMP/E and PAX. Select the installation format that applies to you:
```

Comment:
```suggestion
IBM® SDK for Node.js - z/OS® is available in two installation formats, SMP/E and PAX. Select the installation format that applies to you:
```
@PoojaDurgad it seems your editor isn't accepting the special chars, or my browser isn't displaying them correctly even in 'View file'. Do they appear as they should be for you? If not, can you try to commit the suggested change here to see if it'll work. The line in this editor looks fine in this editor.

Reply:
yes, my editor is not supporting properly .